client/my-sites/checkout/checkout-system-decider.js

```diff
@@ -339,12 +338,6 @@ function getCheckoutVariant(
 		return 'composite-checkout';
 	}
 
-	// Add remaining users to new AB test with 10% holdout
```

Comment:
You can also remove [this conditional](https://github.com/Automattic/wp-calypso/pull/44889/files#diff-ea3634a675c18200b4f625168e835998L325), which just filtered EN/US/USD out of the test cohort. 

You can probably remove [the next conditional](https://github.com/Automattic/wp-calypso/pull/44889/files#diff-ea3634a675c18200b4f625168e835998L336-L342) too, since there's not alternate return after that point, but I guess it doesn't hurt to be explicit for that test.

Reply:
Good point! Removed.